packages/cli/src/cliEntry.js

```diff
@@ -210,6 +212,16 @@ async function setupAndRun() {
   }
 
   logger.setVerbose(commander.verbose);
+
+  const {version: currentVersion} = require(path.join(
+    ctx.root,
+    'node_modules/react-native/package.json',
+  ));
+  const latestRelease = await getLatestRelease(currentVersion);
+
+  if (latestRelease) {
+    printNewRelease(latestRelease, currentVersion);
```

Comment:
this will print at any time, before, during and after running commands. Not cool. It will also print the warning with every command multiple times during the day (depending on how much you use the cli). I think we should show the prompt _at most_ once a day, ideally once per week. This should be a gentle reminder. It's very easy to annoy users with such messages, so we need to be cautious